salesforce-chatbot/client.js

```diff
@@ -0,0 +1,120 @@
+(function () {
+
+    /**
+     * @function buildBindId
+     * @param {Object} context
+     * @description Create unique bind ID based on the campaign and experience IDs.
+     */
+    function buildBindId(context) {
+        return `${context.campaign}:${context.experience}`;
+    }
+
+
+    /**
+     * @function handleChatbot
+     * @param {Object} context
+     * @description calls a function 'embedded_svc.inviteAPI.inviteButton.acceptInvite();'
+     * to activate the chatbot session
+     */
+    function handleChatbot({ context, template }) {
+        embedded_svc.inviteAPI.inviteButton.acceptInvite();
+    }
+
+    /**
+     * @function handleTriggerEvent
+     * @param {Object} context
+     * @description Create trigger event based on context
+     */
+    function handleTriggerEvent({ context, template }) {
+        if (!context.contentZone) return;
+
+        const { userGroup, triggerOptions, triggerOptionsNumber } = context || {};
+
+
+        switch (triggerOptions.name) {
+            case "timeOnPage":
+                return new Promise((resolve, reject) => {
+                    setTimeout(() => {
+                        if (userGroup === "Control") return true;
+
+                        handleChatbot({ context, template });
+                        resolve(true);
+                    }, triggerOptionsNumber);
+                });
+            case "scrollDepth":
+                return SalesforceInteractions.DisplayUtils
+                    .bind(buildBindId(context))
+                    .pageScroll(triggerOptionsNumber)
+                    .then((event) => {
```

Comment:
event is not used anywhere
```suggestion
                    .then(() => {
```